map/navbar.php

```diff
@@ -0,0 +1,45 @@
+<?php function navbar($suffixNavItems = null, $prefixNavItems = null, $additionalItems = null, $brandkLink = "index.php") { ?>
+    
+
+
+<nav class="navbar navbar-expand-md navbar-dark fixed-top" style="background-color: #9F0C35;">
+    <a class="navbar-brand" href="<?php echo $brandkLink ?>" active>
+        <img src="images/logo_white.svg" width="30" height="30" class="d-inline-block align-top" alt="">
+        Railway<strong>Stations</strong>
+    </a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <div class="collapse navbar-collapse" id="navbar">
+        <?php if (!is_null($additionalItems)) {
+            echo $additionalItems;
+        } ?>
+        <ul class="navbar-nav mr-auto order-sm-1 order-md-0">
+            <?php if (!is_null($prefixNavItems)) {
+                echo $prefixNavItems;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$prefixNavItems'.](https://app.codacy.com/app/RailwayStations/RSWebsite/pullRequest?prid=4676723)